tests/b020.py

```diff
@@ -0,0 +1,17 @@
+items = [1, 2, 3]
+
+for items in items:
+    print(items)
+
+items = [1, 2, 3]
+
+for item in items:
+    print(item)
+
+values = {"secret": 123}
+
+for key, value in values.items():
+    print(f"{key=}, {value=}")
+
+for key, values in values.items():
+    print(f"{key=}, {values=}")
```

Comment:
```suggestion
    print(f"{key}, {value}")

for key, values in values.items():
    print(f"{key}, {values}")
```
Are these '=' meant to do something I don't understand in Python?

Reply:
'=' was introduced in [python 3.8](https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging)  
So output would be: `key='secret', value=123`  
Guess better to remove it for backward compatibility 😅